frontend/components/buttons/Button/_styles.scss

```diff
@@ -70,7 +70,6 @@ $base-class: "button";
   border: 0;
   position: relative;
   cursor: pointer;
-  min-width: max-content;
```

Comment:
This was the culprit and accidentally committed 4 months ago. I combed through and saw no reason for the update